cmd/controller/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/awslabs/karpenter/pkg/controllers"
 	"github.com/awslabs/karpenter/pkg/controllers/allocation"
 	"github.com/awslabs/karpenter/pkg/controllers/expiration"
+	"github.com/awslabs/karpenter/pkg/controllers/node"
```

Comment:
I was thinking of "compatibility controller", "node state controller". I actually think that because we're inside the package node, you can refer to other packages named node without colliding. I'd love to revisit this name as we explore how this controller will evolve.